internal/testutils/olm.go

```diff
@@ -60,14 +67,23 @@ func (tc TestContext) AddPackagemanifestsTarget() error {
 	if err != nil {
 		return err
 	}
-	c := &config.Config{}
-	if err = c.Unmarshal(b); err != nil {
+
+	var versioned versionedConfig
+	if err := yaml.Unmarshal(b, &versioned); err != nil {
+		return err
+	}
+	// Create the config object
+	var c config.Config
+	c, _ = config.New(versioned.Version)
```

Comment:
Don't ignore errors that can occur
```suggestion
	c, err := config.New(versioned.Version)
	if err != nil {
		return err
	}
```